diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -734,17 +735,18 @@ fun ASTNode.isGoingAfter(otherNode: ASTNode): Boolean {
 }
 
 /**
- * Get line number, where this node's content starts
+ * Get line number, where this node's content starts. To avoid `ArrayIndexOutOfBoundsException`s we check whether node's maximum offset is less than
```

Comment:
What is the situation when we had this exception? Give code example. 
It looks to be some defence

Reply:
The `Bug1Test.kt` from smoke tests is an example. We insert new code during autofix, but `Document` isn't updated, so maximum offset of the new code exceeds available.